lib/express_validator.js

```diff
@@ -377,7 +384,7 @@ function makeSanitizer(methodName, container) {
     var result;
     this.values.forEach(function(value, i) {
       if (value != null) {
-        var args = [value];
+        var args = [value += ''];
```

Comment:
Similar to the above, it may make more sense to perform the conversion on [line 43](https://github.com/ctavan/express-validator/pull/229/files#diff-6c47d849cc78d146191d4d361699b452L43). Or better yet, DRY and just perform the conversion once for both situations.

Moreover, the assignment here seems unnecessary since the `value` argument is not used further. Though if the desire was to create an assignment for the sake of maintainability, I would do so in its own line, using a new variable so as to not mutate the argument:
``` javascript
strValue += '';
```

Though the first solution is still preferable.